### PR TITLE
[W4.11e][W12-1]Bobbie Ramadhani Soedirgo

### PR DESCRIPTION
**Pull request by soedirgo**

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,28 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // empty list
+        assertNoNull();
```
Comment (asiddharth) [inline test/java/seedu/addressbook/common/UtilsTest.java:50]: Can this function be in-lined here?

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -43,4 +43,28 @@ private void assertAreUnique(Object... objects) {
...
+
+        // no null
+        assertNoNull("a", "b", Integer.valueOf(1));
+        assertNoNull(1, 2, 3, Integer.valueOf(4), 5);
+
+        // some are null
+        assertSomeAreNull((Object) null);
```
Comment (asiddharth) [inline test/java/seedu/addressbook/common/UtilsTest.java:57]: Same comment as above.

**Review by asiddharth — COMMENTED** [review]
Kudos! No major issues with the PR. Good Job @dan-delion

**Comment by soedirgo** [comment]
Will do. Thanks for the review!

**Pull request closed without merging**